MoonDawgArduino/MoonDawgArduino/MoonDawgArduino.ino

```diff
@@ -29,13 +36,24 @@ constexpr char VERTICAL = 'e';
 constexpr char ARM = 'a';
 
 
-// Class constructors
+/* 
+* Class constructors
+* Default constructors are only for "creating" the class 
+* CANNOT HOUSE ANY OF THE SETUP CODE!!
+*/
 DriveMovement driving = DriveMovement();
 DepositMovement depositing = DepositMovement();
 DigMovement digging = DigMovement();
 CameraControl camera = CameraControl();
 
-
+/*
+* This processes the commands sent via the RPi into usuable function calls for the Arduino
+*
+* The function takes a parsed command converts that into two parameters and a command character
+* command is taken into the switch statement and compared to the defined command characters and will use that part of the switch statement
+* this is where either the command will take both params or will use param1 to decide if this is an 'ON' command, where param2
+* decides which of the 'ON' functions will be used.
```

Comment:
don't over explain code, let the code talk

Reply:
It's hard for someone who doesn't know what you did to see it and understand without explanation

Reply:
we should change the code to be more readable then, or create a simpler explanation

Reply:
Probably some of both. Do you want to change that now or later and keep the subpar explanation

Reply:
Change later I guess

Reply:
Let's make something to remind us this needs to be done